### DISCUSSION TITLE
SSL with Windows IIS ReverseProxy

### DISCUSSION THREAD

**Discussion by fault1er1** [kagemomiji/airsonic-advanced#431] [Q&A]
I'm running airsonic-advanced on Windows Server 2019 and I'm curious if I can get airsonic running with an SSL certificate?

By now I start the airsonic server directly from the war file in cmd, just like `"C:\Program Files\Java\jdk-21\bin\java" -Dairsonic.home=C:\\airsonic-kagemomiji\\home -Dserver.port=80 -Xmx1024m -jar "C:\airsonic-kagemomiji\airsonic.war"`.
Does anyone have an idea to get SSL running? Everytime I tried the steps on https://airsonic.github.io/docs/ with reverse proxy and all I'm running in dead ends and gave up :(

Btw, maybe it's possible to implement let's encrypt in airsonic software?

**UPDATE**
Okay, so I managed to get https working with reverse proxy in IIS.

The documentation on https://airsonic.github.io/docs/proxy/iis/ is slightly outdated - or just for Windows Server 2012.

Just install IIS like the documentation says but when it says you need to setup your rewrite rules, add the rule Reverseproxy instead and fill in the required information and that's it. When the server's reaction is to slow don't forget to deactivate compression of static contents.

If someone needs help to set up IIS as reverse proxy for airsonic don't hesitate to ask :D

**Comment by maghiel** [comment]
Great you got it figured out! Normally  I would say: why don't you contribute your findings to the documentation......but it seems dead also. 
@kagemomiji : with the possible merge and new life in airsonic-advanced, perhaps it would be a good idea to fork the original airsonic documentation and maintain it under airsonic-advanced as well?
